### PR TITLE
Fixes not being able to unwrench drawers

### LINKED ISSUE #48970: You can't unwrench drawers
No round id
Reproduction:
-Use wrench on drawer
-put wrench in drawer???
-try again but on disarm intent
-fail
-cry because you can't revive your ai friend on donutstation

### PR DESCRIPTION
**Pull request by Bawhoppen**
:cl:
fix: Drawers & filing cabinets may now unwrench once again.
/:cl:

Fixes: https://github.com/tgstation/tgstation/issues/48970

Sorry, I broke this in https://github.com/tgstation/tgstation/pull/48353.
**150 hours of testing.**

### CODE REVIEW

File: code/modules/paperwork/filingcabinet.dm [hunk 1]
```diff
@@ -46,19 +46,19 @@
 	qdel(src)
 
 /obj/structure/filingcabinet/attackby(obj/item/P, mob/user, params)
-	if(P.w_class < WEIGHT_CLASS_NORMAL)
+	if(P.tool_behaviour == TOOL_WRENCH)
```
Comment (kriskog) [inline code/modules/paperwork/filingcabinet.dm]: Add an intent check here, like on tables that can only be unwrenched if on harm intent. That way you can still insert wrenches.

**Review by ShizCalev — CHANGES_REQUESTED** [review]
what kriskog said.

**Comment by Bawhoppen** [comment]
Sorry I forget about this PR, will do when I get a chance

**Comment by Bawhoppen** [comment]
Should be ready with the suggested changes, although tables can be deconstructed on any intent besides help so I did the same here as well.

**Review by 81Denton — APPROVED** [review]

**Pull request merged**